### ISSUE TITLE
Bad HTML in generated subpanels (Suite7 and SuiteR theme)

### ISSUE THREAD

**Issue by ale26** [salesagility/SuiteCRM#3355]
When sub-panels are generated, the server returns invalid HTML, which results in the browser interpreting it incorrectly.

First subpanel always truncated
![screenshot - 04 04 2017 - 113908](https://cloud.githubusercontent.com/assets/1145477/24648228/acd0e0ba-192b-11e7-9154-1ffd792f307a.png)

Second and other subpanels - is ok
![screenshot - 04 04 2017 - 113922](https://cloud.githubusercontent.com/assets/1145477/24648229/acd10b9e-192b-11e7-8d86-154a41c4b5b2.png)

Problem in file new migrated [include/SubPanel/tpls/SubPanelDynamic.tpl](https://github.com/salesagility/SuiteCRM/commits/master/include/SubPanel/tpls/SubPanelDynamic.tpl)

Smarty template (include/SubPanel/tpls/SubPanelDynamic.tpl) was added to replace xTemplate (include/SubPanel/SubPanelDynamic.html) and pagination block moved incorrectly

include/SubPanel/SubPanelDynamic.html
```
<!-- BEGIN: dyn_list_view -->
<table cellpadding="0" cellspacing="0" width="100%" border="0" class="list view subpanel-table">
<tbody>
<!-- BEGIN: list_nav_row -->
{PAGINATION}

```

include/SubPanel/tpls/SubPanelDynamic.tpl
```
{$PAGINATION}
<table cellpadding="0" cellspacing="0" border="0"  data-empty=".....
```

PAGINATION generate in include/ListView/ListViewSubPanel.php and it is assumed that it is inserted inside the table

```
$html_text = '';
$html_text .= "<tr class='pagination' role='presentation'>\n";
.....
$html_text .= "</tr>\n";
$this->smartyTemplate->assign("PAGINATION", $html_text);
```

#### Your Environment
* SuiteCRM 7.8.2
* Version 56.0.2924.87 (64-bit)

**Comment by rdzsjpr** [comment]
I had a strange issue with the Select\Create button of the **first** subpanel of all Detailviews of SuiteR. My issue was that the subpanel could be dragged within the view independently of the subpanel's Select button. Fixed #3356 helped to solve the issue. I hope your solution is included in the next release. Thank you!

**Comment by ale26** [comment]
This problem also broke the functional of the dynamic reboot of the sub-panel. After the reboot the subpanel doubled and worked incorrectly. This had to find and correct the error. In SuiteP all ok. I've found many similar mistakes, I'll try to find the time and suggest it to the core. It is a pity that the developers no longer support the theme Suite7 and SuiteR.

**Comment by dursuntosun** [comment]
Yes, i have this issue Which is broking the functional of the dynamic reboot of the sub-panel after implement this fix. Please fix this issue because we are using Suite7 theme. Suite7 theme very compact than others.

**Comment by dursuntosun** [comment]
When i try to apply this, it occurs - #3467.

**Comment by ale26** [comment]
@dursuntosun , the problem is known.
I will try to provide a solution within 2 days

**Comment by ale26** [comment]
Fix proposed in issue #3467
